CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+
+<br><br>
+# 🔑 Guidelines for contributing :
+
+Here are some set of guidelines to follow while contributing to `rbjks.github.io` :
+
+1. Welcome to this repository, if you are here as an open-source program participant/contributor.
+2. Participants/contributors have to **comment** on issues they would like to work on, and mentors or the maintainers will assign you.
+3. Issues will be assigned on a **first-come, first-serve basis.**
+4. Participants/contributors can also **open their issues**, but it needs to be verified and labelled by a mentor. We respect all your contributions, whether 
+it is an Issue or a Pull Request.
+5. When you raise an issue, make sure you get it assigned to you before you start working on that project.
+6. Don't create issues that are **already listed**.
+7. Please don't pick up an issue already assigned to someone else. Work on the issues after it gets **assigned to you**.
+8. Create your file in an appropriate folder with appropriate name and extension.
+9. Pull requests will be merged after being **reviewed** by  mentor .
+10. We all are here to learn. You are allowed to make mistakes. That's how you learn, right!.
+
+
+<br>
+
+# ▶️ Steps for contributing :
+## Fork this repository
+<img align="right" width="300"  alt="image" src="https://user-images.githubusercontent.com/110724849/230174676-887cfddf-0c2d-4811-9aa1-0758af091c6b.png">
+
+<br>
+Fork this repository by clicking on the fork button on the top of this page.
+This will create a copy of this repository in your account.
+<br><br><br>
+<br>
+
+## Clone This Repository
+
+<img  align="right" width="200" src="https://user-images.githubusercontent.com/110724849/230175272-c713e91d-f5bf-4cc3-8e37-b36f5d6881fb.png">
+
+Now clone the forked repository to your machine. 
+Go to your GitHub account, open the forked repository, click on the code button and then click the copy to clipboard icon.
+<br>
+
+Open a terminal and run the following git command:
+```
+git clone https://github.com/<Your-Github-username>/rbjks.github.io.git
+```
+<br>
+
+## Create a branch
+
+Change to the repository directory on your computer (if you are not already there):
+
+```
+cd rbjks.github.io
+```
+
+Now create a branch using the `git checkout` command:
+
+```
+git checkout -b <branch-name>
+```
+
+For example:
+
+```
+git checkout -b add-new-feature
+```
+
+## Make necessary changes and commit those changes
+If you go to the project directory and execute the command `git status`, you'll see there are changes.
+
+Add those changes to the branch you just created using the git add command:
+
+```
+git add .
+```
+Now commit those changes using the git commit command:
+
+```
+git commit -m "Your commit message"
+```
+## Push changes to GitHub
+Push the changes to the forked repository using:
+```
+git push origin <Your-branch-name>
+```
+Navigate to the original repository and make a pull request.
```

Comment:
Might want to expand on how to create a PR

Reply:
Sure will do that